references/backup.md

```diff
@@ -0,0 +1,58 @@
+# Backup
+
+The `backup` route allow user to trigger a backup creation and download created backup.
+
+## Trigger backup
+
+<RouteHighlighter method="POST" route="/backup"/>
+
+Trigger a backup creation process
+
+### Example
+
+<code-samples id="post_backup_1" />
+
+#### Response: `202 Accepted`
+
+```json
+{
+    "id": 5
+}
+```
+
+## Get backup status
```

Comment:
Why is this not a common `update` with an `updateId` ? Or like @curquiza asked, the ID that is given back on trigger an `updateId` ?
In which case this route seems a bit redundant.